packages/graphql/src/translate/queryAST/ast/selection/RelationshipSelection.ts

```diff
@@ -81,4 +81,14 @@ export class RelationshipSelection extends EntitySelection {
             selection: match,
         };
     }
+
+    protected getRelationshipDirection(): "left" | "right" | "undirected" {
+        return this.relationship.getCypherDirection(this.directed);
+    }
+}
+
+export class DirectedRelationshipSelection extends RelationshipSelection {
```

Comment:
It may be worth a comment

```suggestion
/** Enforces direction on the relationship selection, regardless of direction configuration **/
export class DirectedRelationshipSelection extends RelationshipSelection {
```